niworkflows/interfaces/cifti.py

```diff
@@ -422,3 +428,40 @@ def _create_cifti_image(bold_file, label_file, bold_surfs, annotation_files, tr,
     out_file = "{}.dtseries.nii".format(split_filename(bold_file)[1])
     ci.save(img, out_file)
     return Path.cwd() / out_file
+
+
+def _reorient_image(img, orient_img, orient_target):
+    """
+    Coerce an image to a target orientation.
+
+    .. note::
+        Only RAS -> LAS conversion is currently supported
+
+    Parameters
+    ----------
+    img : `spatialimage`
```

Comment:
I'd be nice to figure out the right reference for this intersphinx source.
```suggestion
    img : :obj:`spatialimage`
```